device_doctor/README.md

```diff
@@ -0,0 +1,72 @@
+# device_doctor
+
+This utility tool is used by LUCI infrastructure to manage device health
+for Flutter LUCI swarming bots.
+
+It offers support for different platforms: linux, mac and windows, and
```

Comment:
platforms->host platforms is clearer as Android and iOS are platforms too.

Reply:
Done.